kubernetes/hapi-deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         app: monitor-hapi
     spec:
       imagePullSecrets:
-          - name: regcred
+        - name: regcred
```

Comment:
@xavier, did you check *all* the deployment files? I don't see the webapp ones for prod, dev, nor postgres.